bindings/java/iota-wallet-java/.gitignore

```diff
@@ -1,6 +1,9 @@
 .gradle
 gradle.properties
 lib/build
+lib/bin
```

Comment:
why is this one needed / when is this folder created?

Reply:
Its beeing made by my vscode java plugin